DANS-KNAW/dd-verify-migration
# DD-727: Predict dataverse files from the vault

Fixes DD-727: Predict dataverse files from the vault

# Description of changes

Read BagInfo and BagSequence from dd-bag-index and manifests from dd-bag-store to created record in the expected table.

* [x] CLI should not interpret config.yml also(!) as CSV

# How to test

* start the  VMs deasy and dar
* Load bags in the vault (in case of a fresh deasy), for example from the root of `dd-sword2-exmples`:

       ./run.sh SequenceSimple https://deasy.dans.knaw.nl/sword2/collection/1 user001 user001 src/main/resources/agreement-flow/valid/revision*
       ./run.sh Simple https://deasy.dans.knaw.nl/sword2/collection/1 user001 user001 src/main/resources/agreement-flow/valid/XXX

* collect UUIds from the log or from the table `bag_info` in `DB easy_bag_index`
* from the root of  `dd-verify-file-migration`:
  put the UUIDs in `etc/uuids.txt` and execute in two terminals

      rm -rf data/*;start-hsqldb.sh

      mvn clean install
      start.sh load-from-vault -u etc/uuids.txt

* [x] or build, deploy on deasy and execute

       dd-verify-file-migration load-from-vault -u /vagrant/shared/uuids.txt

# Related PRs

* ...

# Notify
@DANS-KNAW/dataversedans
